src/index.js

```diff
@@ -37,6 +37,8 @@ export { Button                  } from './atoms/Button';
 export { BylineImage             } from './atoms/BylineImage';
 export { BylineName              } from './atoms/BylineName';
 export { BylineWithTwoLines      } from './atoms/BylineWithTwoLines';
+export { Description             } from './atoms/MainRecipe/Description';
```

Comment:
I think we will be better off with removing some semantics from this one – how about `SplashText` or `ColorTextBox`?